tests/renderer/components/__snapshots__/settings-general-appearance-spec.tsx.snap

```diff
@@ -63,11 +63,19 @@ exports[`AppearanceSettings component renders 1`] = `
       </a>
       . The easiest way to get started is to clone one of the two existing themes and to add your own colors.
     </p>
+    <p>
+      Additionally, if you wish to import an editor theme from VSCode, just pick your JSON file and Fiddle will do the rest.
```

Comment:
In its current state, we should probably mention that this is a feature for importing from Monaco rather than a VSCode JSON file, since the latter can't be imported directly.